src/test/java/seedu/address/commons/util/CollectionUtilTest.java

```diff
@@ -5,13 +5,12 @@
 import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
 import static seedu.address.testutil.Assert.assertThrows;
 
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.Collections;
-import java.util.List;
+import java.util.*;
```

Comment:
Should we list out all the imports instead of using a wildcard?

Reply:
Thank you for pointing that out. I have changed it.